internal/comparetest/testdata/traces/ignore-one-resource-attribute/actual.json

```diff
@@ -0,0 +1,147 @@
+{
+    "resourceSpans": [
+        {
+            "resource": {
+                "attributes": [
+                    {
+                        "key": "testKey1",
+                        "value": {
+                            "stringValue": "teststringvalue1"
+                        }
+                    }
+                ],
+                "droppedAttributesCount": 0
+            },
+            "scopeSpans": [
+                {
+                    "spans": [
+                        {
+                            "attributes": [
+                                {
+                                    "key": "testKey1",
```

Comment:
Please don't use the same attribute in trace and resource. It's invalid data

Reply:
I change the attribute key of resource into `host.name`